setup.cfg

```diff
@@ -9,7 +9,7 @@ python_files = *Test.py
 python_classes = *Test
 python_functions = *_test
 timeout = 35
-addopts = --color=yes
+addopts = --color=yes --ignore=test-requirements.txt
```

Comment:
I was wondering the same. But I can't find any other method of doing it :o
Can you ?

On Tue, Mar 1, 2016 at 5:13 PM, Abhay Raizada <notifications@github.com>
wrote:

> In setup.cfg
> <https://github.com/coala-analyzer/coala-bears/pull/134#discussion_r54555781>
> :
>
> > @@ -9,7 +9,7 @@ python_files = *Test.py
> >  python_classes = *Test
> >  python_functions = *_test
> >  timeout = 35
> > -addopts = --color=yes
> > +addopts = --color=yes --ignore=test-requirements.txt
>
> and are you sure this how we ignore files?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/134/files#r54555781>.
>